pitop/processing/utils/vision_functions.py

```diff
@@ -1,3 +1,12 @@
+
+try:
+    import cv2
+except (ImportError, ModuleNotFoundError):
+    raise ModuleNotFoundError(
+        "OpenCV Python library is not installed. You can install it by running "
+        "'sudo apt install python3-opencv libatlas-base-dev'.") from None
```

Comment:
lets not do this

Reply:
Let's do what then?

Reply:
Agree - revert this and leave the imports